mutiny-core/src/federation.rs

```diff
@@ -560,6 +560,7 @@ impl<S: MutinyStorage> FederationClient<S> {
         let mut stored_payment: MutinyInvoice = invoice.clone().into();
         stored_payment.inbound = inbound;
         stored_payment.labels = labels;
+        stored_payment.status = HTLCStatus::InFlight;
```

Comment:
nice catch. Though we don't typically use `InFlight` much, just wondering if it was `Pending` before and was being filtered out for whatever reason, or really why `InFlight` is used at all.

Reply:
`Pending` is what we use for invoices we create that are unpaid